pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/retrieve/activities/ActivityDefinitionsRetriever.java

```diff
@@ -0,0 +1,30 @@
+
+package org.broadinstitute.dsm.model.elastic.retrieve.activities;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+import org.broadinstitute.dsm.db.DDPInstance;
+import org.broadinstitute.dsm.model.elastic.retrieve.ElasticSearchObjectsRetriever;
+import org.broadinstitute.dsm.util.ElasticSearchUtil;
+
+public class ActivityDefinitionsRetriever implements ElasticSearchObjectsRetriever<Map<String, Map<String, Object>>> {
+
+    private final DDPInstance instance;
+
+    public ActivityDefinitionsRetriever(DDPInstance instance) {
+        this.instance = instance;
+    }
+
+    @Override
+    public Map<String, Map<String, Object>> retrieve() {
+        Map<String, Map<String, Object>> activityDefinitions =
+                new ConcurrentHashMap<>(ElasticSearchUtil.getActivityDefinitions(instance));
```

Comment:
I think it is good idea to move `getActivityDefinitions` method to `ElasticSearchable` and `ElasticSearch` to eliminate `ElasticSearchUtil` and `DDPInstance` usage.

Reply:
@gmakharat I agree with defining this method in interface and making it as a `default` implementation / implementing in `ElasticSearch`. 

However there's one problem with removing `DDPInstance`. If you look at `DisplaySettingsRoute.java` you will see that `DDPInstance` is used in many places.

So I belive I have two options:
  - use `DDPInstance` as a dependency for the new method in `ElasticSearchable`
  - use `DDPInstanceDto` as a dependecy for the new method in `ElasticSearchable` and refactor the whole `DisplaySettingsRoute.java`
 
I would like to know your point of view on this, thxs!

Reply:
depends how widely `DDPInstance` is used in `DisplaySettingsRoute`, if it is easy change, the it worths to change `DDPInstance` to `DDPInstanceDto`, if not, let's keep it how it is right now.